Addressables/Editor/FileToAddressableGroupTempCache.cs

```diff
@@ -0,0 +1,202 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using UnityEditor;
+using UnityEditor.AddressableAssets.Settings;
+using UnityEditor.AddressableAssets.Settings.GroupSchemas;
+using UnityEngine;
+
+namespace Com.Innogames.Core.Frontend.NodeDependencyLookup.Addressables
+{
+	public class FileToAddressableGroupDependency
+	{
+		public const string Name = "FileToAddressableGroup";
+	}
+
+	public class FileToAddressableAssetGroupTempCache : IDependencyCache
```

Comment:
There is a lot of similar code in all implementations of IDependencyCache. Maybe implement a common base class

Reply:
I am not sure if this will improve much. At first they look quite similar but they each have a lot of differences in the implementation.